src/evm/protocol/uniswap_v4/state.rs

```diff
@@ -712,6 +712,33 @@ impl ProtocolSim for UniswapV4State {
             }
         }
 
+        // Update block information if provided in the delta attributes
+        if let Some(block_number_bytes) = delta
```

Comment:
Is it dangerous if we only update the block_number and not the timestamp?

Reply:
Yes, pls raise a warning if one is missing, please - so we know something is funky